templates/managed-ubuntu.yaml

```diff
@@ -497,9 +497,9 @@ Resources:
           - - |
               #!/bin/bash -xe
             - |
-              apt-get update
+              apt update
```

Comment:
on the ubuntu linux command line i believe it's actually `apt-get -y`.  the `-y` flag is to not prompt for confirmation.

Reply:
maybe i'm wrong.